libs/cardano-protocol-tpraos/CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Version history for `cardano-protocol-tpraos`
 
-## 1.0.3.8
+## 1.1.0.0
 
-*
+* Change teh type of `bsize` and `hBbsize` to `Word32`
```

Comment:
Small typo here

Reply:
Yeah, I noticed that too. I'll fix that at some later point. I already have a release ready for this: https://github.com/IntersectMBO/cardano-haskell-packages/pull/645

Reply:
Fixed.